uberfire-extensions/uberfire-wires/uberfire-wires-core/uberfire-wires-core-grids/src/main/java/org/uberfire/ext/wires/core/grids/client/widget/grid/renderers/grids/impl/BaseGridRendererHelper.java

```diff
@@ -245,9 +287,13 @@ public RenderingInformation getRenderingInformation() {
                 }
             }
         }
+        if (LOGGER.isLoggable(Level.FINEST)) {
```

Comment:
@manstis 
Pls remove all that duplicated Logging code

Reply:
Done.